### ISSUE TITLE
NEW item variant: old redirection cube

### ISSUE THREAD

**Issue by SCIENTIST2004** [BEEmod/BEE2-items#2807]
![old lc](https://user-images.githubusercontent.com/35931028/44235476-5bba8080-a1b2-11e8-8007-e1aaa989ac73.png)
![blue print lc](https://user-images.githubusercontent.com/35931028/44235480-607f3480-a1b2-11e8-825d-0bba118555e9.png)

**Comment by SCIENTIST2004** [comment]
edgeless safety cube
![576909080-612x612](https://user-images.githubusercontent.com/35931028/44267033-623e0c00-a235-11e8-81dc-7b4b22f91cd0.jpg)

**Comment by SCIENTIST2004** [comment]
![futbol](https://user-images.githubusercontent.com/35931028/44281340-253b3f00-a260-11e8-9865-7e0403a14cdf.png)

**Comment by SCIENTIST2004** [comment]
![futbol](https://user-images.githubusercontent.com/35931028/44281810-b232c800-a261-11e8-9575-98df7266755e.png)

**Comment by ZipZoptheCat** [comment]
What about franken cubes?

**Comment by SCIENTIST2004** [comment]
![old fc](https://user-images.githubusercontent.com/35931028/44287840-fbd9dd80-a276-11e8-99bf-935d1bb157fb.png)

**Comment by King0191** [comment]
We can't change Franken Turret models tho…? Isn't that a THING?

**Comment by HugoBDesigner** [comment]
I'm afraid we can't change Franken Turrets.

**Comment by TeamSpen210** [comment]
And someone will need to create these models, and texture them.

**Comment by HugoBDesigner** [comment]
While I'm aware the goal is to keep these models simple, I think they're a bit *too* simple. Additional details and familiar aspects would make them more appealing than, say, a generic metal ball.

**Comment by TeamSpen210** [comment]
The other problem with the ball is that we can't build cubemaps, which will make it really noticable with a highly-reflective object that doesn't actually reflect the surroundings.

**Comment by ZipZoptheCat** [comment]
Does beemod have an in house modeler currently? And what tools are needed to model?

**Comment by Endermage77** [comment]
The answers to those are "No" and "Pretty much any modelling software."

**Comment by ZipZoptheCat** [comment]
I've been trying my hand at the khed modeling program but I'm going to need textures. Does anyone have the textures for 4 by 4 grid portallable and non-portallable walls? Or any other useful textures.

**Comment by SCIENTIST2004** [comment]
Auto-Portal
![auto portal](https://user-images.githubusercontent.com/35931028/44956205-eeecf900-aec8-11e8-9bb3-98ac23dc4105.png)

**Comment by Galaxy5151** [comment]
Old edgeless safety cube:
![sphere thingy](https://user-images.githubusercontent.com/35960636/45980929-448d6f00-c04b-11e8-87b9-0ad3e89abb78.png)

**Comment by Galaxy5151** [comment]
Edgeless safety cube:
![sphere thingy](https://user-images.githubusercontent.com/35960636/45981036-9f26cb00-c04b-11e8-8d14-5d7817177204.png)

**Comment by Galaxy5151** [comment]
Bomb cube:
![cubething](https://user-images.githubusercontent.com/35960636/45982893-7c4be500-c052-11e8-8d0f-d606106a360f.png)

**Comment by DrMeepster** [comment]
There could be a workaround for the frankencube by changing the render mode of the franken turret and then parenting the new model to it.

**Comment by TeamSpen210** [comment]
Not really. For one then we wouldn't be able to know when exactly the turret is animating, to match the movement. We also wouldn't be able to alter the appearance for gels. When held partially through portals, it'd only appear on one side, and it'd either disappear entirely when held in Coop or appear as the normal one.

**Comment by Endermage77** [comment]
Luckily for you, you don't need to worry about gels as they don't affect franken turrets for who knows why.

The only effect gels have on franken turrets is that bouncy gel turns the franken turret into a trampoline the player can bounce on. Bouncy Frank Jumping is a skill that is very hard to master.

**Comment by Konclan** [comment]
Once Salt Mines comes out we have full permission to use their old aperture stuff
